cpu/native/native_cpu.c

```diff
@@ -225,7 +225,7 @@ void thread_yield_higher(void)
         irq_enable();
     }
     else {
-        isr_thread_yield();
+        sched_context_switch_request = 1;
```

Comment:
I think `isr_thread_yield` is not called from anywhere else - please remove if this is correct.